bench/chromium/serve_guardian.py

```
@@ -0,0 +1,235 @@
+        status = shell_status(child.wait())
+        completed = wait_for_completion(
+            completion_path, completion_token, os.getpgrp())
+        if not completed and status == 0:
+            print(
+                "FAILED: replay lease guardian: command exited 0 before "
+                "a root supervisor armed completion",
+                file=sys.stderr,
+                flush=True,
+            )
+            status = 125
+    except BaseException as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `BaseException`

(BLE001)